components/CircleChart.js

```diff
@@ -0,0 +1,10 @@
+import React, { useContext } from 'react';
+
+export default function checkContext() {
+  const { sharedState } = useContext()
+  console.log('dawg', sharedState)
```

Comment:
Was just messing around with some data here, this whole file is just random code. It will be replaced with the actual CircleChart component in the next merge